worker/dbaccessor/worker_cgo.go

```diff
@@ -85,8 +85,8 @@ func (a *dqliteApp) Close() error {
 	return a.dqliteApp.Close()
 }
 
-// Repl returns a Repl worker from the underlying DB.
-func (a *dqliteApp) Repl(dbGetter DBGetter) (REPL, error) {
+// GetRepl returns a Repl worker from the underlying DB.
+func (a *dqliteApp) GetRepl(dbGetter DBGetter) (REPL, error) {
```

Comment:
minor nit: `getREPL` as it is an acronym ;-)

Reply:
whoops!